src/Symfony/Component/DependencyInjection/Tests/EnvVarProcessorTest.php

```diff
@@ -956,4 +967,5 @@ public static function provideGetEnvDefined(): iterable
         yield 'Null' => [false, fn () => null];
         yield 'Env var not defined' => [false, fn () => throw new EnvNotFoundException()];
     }
+
```

Comment:
Please apply the patch provided by fabbot in the CI.

Reply:
If you were talking about the empty line I have  removed it.

Reply:
> [Review](https://fabbot.io/report/symfony/symfony/52369/b792342c6f24f3d04f8f860a237aa61d1ef30d05#patchcs) the proposed patch then [download](https://fabbot.io/patch/symfony/symfony/52369/b792342c6f24f3d04f8f860a237aa61d1ef30d05/cs.diff) it to apply it manually or execute the following command from the repository root directory:

```console
curl https://fabbot.io/patch/symfony/symfony/52369/b792342c6f24f3d04f8f860a237aa61d1ef30d05/cs.diff | patch -p0
```